CONTRIBUTING.md

```diff
@@ -31,123 +31,112 @@ Note that we use the Google CLA because the OpenConfig project is
 [administered and maintained by Google](https://opensource.google.com/docs/cla/#why),
 not to ascribe any specific rights to a single OpenConfig contributor.
 
-## Make a contribution
-
-To make a contribution to OpenConfig featureprofiles:
-
-1.  Open a pull request in the
-    [openconfig/featureprofiles](https://github.com/openconfig/featureprofiles)
-    repo. A brief description of the proposed addition along with references to
-    any discussion issues should be included.
-
-    *   Pull requests should be kept small. An ideal change is less than 500
-        lines. Small changes allow detailed discussions of the additions that
-        are being made to the model, whilst also ensuring that
-        course-corrections can be made early in the process. If a test is
-        growing to more than 500 lines, it may need to be broken into multiple
-        smaller tests.
-    *   New contributions should be in the feature/experimental directory.
-
-1.  The pull request should include both the suggested feature profile textproto
-    additions, as well as any relevant additions to tests. Tests should be
-    written in golang using the [ONDATRA](https://github.com/openconfig/ondatra)
-    framework.
-
-1.  The automated CI running against each pull request will check the pull
-    request for compliance. The author should resolve any issues found by CI.
-
-1.  One or more peers in the community may review the pull request.
+## Code Style
 
-1.  A feature profile repository maintainer will be reponsible for a final
-    review and approval. Only a feature repository maintainer can merge a pull
-    request to the main branch.
+All code should follow
+[Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
+and [Effective Go](https://go.dev/doc/effective_go) for writing readable Go code
+with a consistent look and feel.
 
-The aim of this process is maintain the model quality and approach that
-OpenConfig working group has strived for since its inception in 2014. Thank you
-for your contributions!
+All code and documentation should follow
+[Google developer documentation style guide](https://developers.google.com/style/word-list)
+for the use of inclusive language.
 
-## Test development style guide
+It is recommended that tests follow
+[Testing on the Toilet](https://testing.googleblog.com/) for best practices on
+design patterns.
 
-### Directory Organization
+## Directory Organization
 
 The directory tree is organized as follows:
 
 *   `cloudbuild/` contains google cloud build scripts for running virtual
     routers in containers on [KNE](https://github.com/google/kne)
 *   `feature/` contains definition and tests of feature profiles.
 *   `feature/experimental` contains new features and tests which are not yet
-    proven to be stable
+    categorized or not confirmed to pass.
```

Comment:
Should state how a test is moved from experimental and into feature/

Reply:
Generally, after they are categorized and confirmed to pass, but I think we may want to reserve our discretion on a case by case scenario.